src/languages/elixir.md

```diff
@@ -0,0 +1,127 @@
+# Elixir
+
+Platform.sh supports building and deploying applications written in Elixir.  They are automatically compiled during the build phase, and support both committed dependencies and download-on-demand. The underlying Erlang version is 22.0.7.
+
+## Supported versions
+
+* 1.9
+
+To specify an Elixir container, use the `type` property in your `.platform.app.yaml`.
+
+{% codesnippet "/registry/images/examples/full/elixir.app.yaml", language="yaml" %}{% endcodesnippet %}
+
+## Assumptions
+
+Platform.sh is making assumptions about your application to provide a more streamlined experience. These assumptions are the following:
+
+- Your `mix.exs` file is at the root of your application.
+
+Platform.sh will then run `mix do deps.get, deps.compile, compile` on your application to build a binary.
```

Comment:
Is it important to note that it installs rebar? I guess by calling `mix local.rebar`
https://lab.plat.farm/images/elixir/-/blob/1.9/files/etc/platform/flavor.d/default.py#L43

Reply:
I added a note about the default build